common/src/main/java/com/tc/classloader/ServiceLocator.java

```diff
@@ -116,4 +115,44 @@
     return implementations;
   }
 
```

Comment:
Let's try and explore some ways we can do this part differently.  Using system properties and directory scanning may be OK in the interim but I'd like to find a more sustainable solution.